dlrover/python/common/constants.py

```diff
@@ -238,8 +238,14 @@ class NodeEnv(object):
     RANK = "RANK"  # It is the rank of node not the rank of process.
     WORLD_SIZE = "WORLD_SIZE"  # It is the number of nodes.
 
-    # process env
+    # worker process env
     TORCHELASTIC_RUN_ID = "TORCHELASTIC_RUN_ID"
+    TORCHELASTIC_RESTART_COUNT = "TORCHELASTIC_RESTART_COUNT"
```

Comment:
Remove the unused constant (to avoid extensive changes for subsequent torch compatibility), and also need comments to indicate that it is bound to the torch implementation.

Reply:
ok